docs/src/pages.ts

```diff
@@ -1,20 +1,37 @@
-import type { MuiPage, OrderedMuiPage } from 'docs/src/MuiPage';
-import standardNavIcons from 'docs/src/modules/components/AppNavIcons';
+import type { MuiPage } from 'docs/src/MuiPage';
 
 const pages: readonly MuiPage[] = [
   { pathname: '/versions' },
+  { pathname: '/docs/callouts', title: 'Callouts' },
   {
-    pathname: 'https://mui.com/store/',
-    title: 'Templates',
-    icon: standardNavIcons.ReaderIcon,
-    linkProps: {
-      'data-ga-event-category': 'store',
-      'data-ga-event-action': 'click',
-      'data-ga-event-label': 'sidenav',
-    },
+    pathname: '/about',
+    title: 'Main parent',
+    children: [
+      {
+        pathname: '/x/react-data-grid-group',
+        title: 'First-level child',
+        children: [
+          { pathname: '/x/react-data-grid/editing', title: 'Second-level child' },
+          { pathname: '/x/react-data-grid', title: 'Second-level child' },
+          {
+            pathname: '/x/api/data-grid-group',
+            title: 'Second-level child parent',
```

Comment:
This one look a bit strange (same on X docs), due to the expand button it do not align with other pages of the same depth.

![image](https://github.com/mui/material-ui/assets/45398769/adfc1e94-82e1-4e12-80f6-7b7dfd5332d5)


I think we have a distinct behavior for first level parts (icon on the left), and their children (icon on the right)

Here is quick CSS hack to test, but might be better to directly modify the component

```css
.MuiCollapse-wrapper .MuiLink-root .ItemButtonIcon { order: 2; }
```
    
![image](https://github.com/mui/material-ui/assets/45398769/84b37923-454c-4e70-9896-6e2d11d3c97b)

Reply:
Feel free to push these changes! I think the screenshot already is a step forward and we can iterate from there! It was, at least, what I was intending to do already ⎯ something similar to what you've done there.

Reply:
I would do I I could, but GitHub says I don't have access to Andrew's branch, so that's the best I managed to share 😅